bwmenu

```diff
@@ -247,30 +277,41 @@ clipboard-wayland-clear() {
 # copy to clipboard and give the user feedback that the password is copied
 # $1: json item
 copy_password() {
-  pass=$(echo "$1" | jq -r '.login.password')
+  if not_unique "$1"; then
+    ITEMS=$1
```

Comment:
shouldn't this be an array of items in stead of only one?

Reply:
We're talking about L278 right? Yes, always an array of items.